next/components/forms/segments/AccountSectionHeader/TaxFeeSectionHeader.tsx

```diff
@@ -14,6 +14,7 @@ import Button from '../../simple-components/Button'
 type AccountSectionHeaderBase = {
   title: string
   status?: string
+  who?: string
```

Comment:
we've talked about this in previous PR , but if you must then at least

```suggestion
  // TODO temp only for testing, remove the prop once server integration is ready
  who?: string
```